src/_data/sidenav.yml

```diff
@@ -63,34 +65,38 @@
             - title: Hero animations
             - title: Staggered animations
         - title: Advanced UI
+          permalink: /development/ui/advanced
           children:
             - title: Focus handling system # Where did this go?
             - title: Gestures
               permalink: /development/ui/advanced/gestures
         - title: Platform-specific theming
         - title: Widget catalog
-          permalink: /development/ui/widgets-catalog
-    - title: Data and backend
+          permalink: /development/ui/widget-catalog
+    - title: Data & backend
+      permalink: /development/data-and-backend
       children:
         - title: Architect app to manage data flow
         - title: JSON and serialization
           permalink: /development/data-and-backend/json
         - title: Back-end services
         - title: Use Firebase with Flutter
         - title: Run background processes
-    - title: Accessibility & internationalization
-      permalink: /accessibility
+    - title: Accessibility & internationalization # TODO: use: Accessibility & localization
+      permalink: /development/accessibility-and-localization
```

Comment:
There is a note in the IA that this will eventually be "Accessibility & localization", but despite that, I've used `/development/accessibility-and-localization` as the path. Is that ok @Sfshaza @galeyang?

Reply:
Yes

Reply:
LGTM.